example/scalalib/publishing/9-cosmo-assembly/build.mill

```diff
@@ -0,0 +1,20 @@
+//// SNIPPET:BUILD
+package build
+import mill._, scalalib._
+
+object foo extends ScalaModule with cosmo.CosmoModule {
+  def scalaVersion = "2.13.16"
+}
+//// SNIPPET:END
+
+// This example uses the `scalalib.cosmo.CosmoModule` to build a cross-platform executable assembly by prepending
+// a launcher in the Actually Portable Executable (APE) format compiled with the Cosmopolitan's toolchain.
```

Comment:
This explanation needs to be fleshed out. What does cross-platform mean? What platforms does it support? How does it work? Where can a user go to read more about this?